bgpd/bgp_mplsvpn.c

```diff
@@ -963,17 +963,17 @@ DEFUN (show_bgp_ip_vpn_rd,
        "VPN Route Distinguisher\n"
        JSON_STR)
 {
-  int idx_ext_community = 5;
+  int idx_rd = 5;
```

Comment:
Additionally, removing the 'ip' serves only to fragment the command space. I went through and merged the entire `show bgp` CLI tree so that 'ip' may be optionally prepended some time ago.